test/test_translate.py

```diff
@@ -93,3 +93,82 @@ def testORF1abPolyprotein(self):
         # K   'repeats' times, then  N   L   N   P   L   K
         expected = 'K' * repeats + 'NLNPLK'
         self.assertEqual(expected, translate(seq, 'ORF1ab polyprotein'))
+
+
+class TestTranslateSpike(TestCase):
+    """
+    Tests for the translate.translateSpike function.
+    """
+    def testNoGapsCorrectLength(self):
+        """
+        A sequence with no gaps must have the correct length.
+        """
+        seq = 'TTGGTTGTTTATTACCAC'
+        self.assertEqual(len(seq) / 3, len(translateSpike(seq)))
```

Comment:
Why do you test the lengths here? If you just put in the expected AA sequence (either by hand or using `from Bio.Seq import Seq` (as in translate.py) then you could test the length and the actual sequence at once.

Reply:
Done.